### PR TITLE
Rework sum-of-multiples documentation

### PR DESCRIPTION
**Pull request by kytrinyx**
**This exercise is one of the focus exercises for 12in23.**

This PR is part of our project of making our Practice Exercises more consistent and human. For more context please [see this forum thread](https://forum.exercism.org/t/new-project-making-practice-exercises-more-consistent-and-human-across-exercism/3943)

We aim to merge this PR in roughly 72 hours, and then submit PRs to all tracks that have this exercise.

The main change is to frame the exercise within the context of a story.

### CODE REVIEW

**Comment by IsaacG** [comment]
To be honest, I find the old prose a lot easier to read. It surfaces the numbers/examples/details without having to parse all the prose. Would it make sense to have an example that is more numbers and less text?

**Comment by kytrinyx** [comment]
Do you have any suggestions for a pretext that adds color without all the verbosity? I really struggled with this one because I couldn't find a single down to earth, real world situation that would use this.

I don't like the old one at all, but I would be open to just deprecating the exercise if we can't find a workable story.

**Comment by IsaacG** [comment]
I don't have a story suggestion and I'm perfectly fine with this one.

I think, in general, examples are much easier to read when they focus on the values and use less prose. Mixing the prose and example data makes the example much harder to follow and adds another layer of "word problem" reading comprehension. 

```diff
- To find the unique multiples, we can combine these two sets of multiples and remove any duplicates: `{3, 5, 6, 9, 10, 12, 15, 18, 20}`
- The sum of all these unique multiples is: `3 + 5 + 6 + 9 + 10 + 12 + 15 + 18 + 20 = 98`
+ Find the unique multiples (combing the sets and removing duplicates): `{3, 5, 6, 9, 10, 12, 15, 18, 20}`
+ Sum the unique multiples: `3 + 5 + 6 + 9 + 10 + 12 + 15 + 18 + 20 = 98`
```

**Comment by kytrinyx** [comment]
> examples are much easier to read when they focus on the values and use less prose

This is a really useful observation, and I'm adding it to my personal list of guidelines to follow when working on these.

I like your improvement here.

**Comment by kytrinyx** [comment]
What do you think of this?

```diff
-- The player completed level 20 and found two magical items with base values of 3 and 5.
-- To calculate the energy points earned by the player, we need to find all the unique multiples of these base values that are less than or equal to level 20.
-- For the base value of 3, the multiples less than or equal to 20 are: 3, 6, 9, 12, 15, 18, and 20.
-- For the base value of 5, the multiples less than or equal to 20 are: 5, 10, 15, and 20.
-- To find the unique multiples, we can combine these two sets of multiples and remove any duplicates: `{3, 5, 6, 9, 10, 12, 15, 18, 20}`
-- The sum of all these unique multiples is: `3 + 5 + 6 + 9 + 10 + 12 + 15 + 18 + 20 = 98`
+**The player completed level 20 and found two magical items with base values of 3 and 5.**
+
+To calculate the energy points earned by the player, we need to find all the unique multiples of these base values that are less than or equal to level 20.
+
+- Multiples of 3 up to 20: `{3, 6, 9, 12, 15, 18, 20}`
+- Multiples of 5 up to 20: `{5, 10, 15, 20}`
+- Combine the sets and remove duplicates: `{3, 5, 6, 9, 10, 12, 15, 18, 20}`
+- Sum the unique multiples: `3 + 5 + 6 + 9 + 10 + 12 + 15 + 18 + 20 = 98`
 - Therefore, the player earns **98** energy points for completing level 20 and finding the two magical items with base values of 3 and 5.
```

**Comment by IsaacG** [comment]
> What do you think of this?

I think that's a lot simpler to read :) Thanks!

**Comment by kytrinyx** [comment]
Thanks @IsaacG. I've pushed that up!

**Review by ErikSchierboom — APPROVED** [review]

**Review by ErikSchierboom — APPROVED** [review]

File: exercises/sum-of-multiples/instructions.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- Remove any duplicates.
+- Calculate the sum of all the numbers that are left.
+
+Let's look an example;
+
+**The player completed level 20 and found two magical items with base values of 3 and 5.**
+
+To calculate the energy points earned by the player, we need to find all the unique multiples of these base values that are less than or equal to level 20.
+
+- Multiples of 3 up to 20: `{3, 6, 9, 12, 15, 18, 20}`
```
Comment (petertseng) [inline exercises/sum-of-multiples/instructions.md:23]:
```suggestion
- Multiples of 3 up to 20: `{3, 6, 9, 12, 15, 18}`
```

since 20 is not a multiple of 3, I think it is necessary to remove 20 from this list. The only other reason I could think of to include 20 is that the upper limit should always be included, but nothing seems to indicate that.

Comment (kytrinyx) [reply exercises/sum-of-multiples/instructions.md:23]: You're absolutely correct.  This was totally just a messed up edit on my part. Thank you for catching it!

File: exercises/sum-of-multiples/instructions.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Instructions
+
+Your task is to write the code that calculates the energy points that get awarded to players when they complete a level.
+
+The points awarded depend on two things:
+
+- the number of the level that the player completed
```
Comment (SleeplessByte) [inline exercises/sum-of-multiples/instructions.md:7]:
```suggestion
- the level (as number) that the player completed
```

I think from the rest of the exercise, the level _is_ numeric and the number is not a property of the level. If it is, then the sentence is, for me, hard to parse.

A bit futher down it says "level number" which also works, but is less suited for this sentence.

File: exercises/sum-of-multiples/instructions.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- the base value of each magical item collected by the player during that level
+
+The energy points are awarded according to the following rules:
+
+- For each magical item, take the base value and find all the multiples of that value that are less than or equal to the level number.
+- Combine the sets of numbers.
+- Remove any duplicates.
+- Calculate the sum of all the numbers that are left.
```
Comment (SleeplessByte) [inline exercises/sum-of-multiples/instructions.md:15]: These read like steps, not arbitrary bullet points or rules.

```suggestion
1. For each magical item, take the base value and find all the multiples of that value that are less than or equal to the level number.
2. Combine the sets of numbers.
3. Remove any duplicates.
4. Calculate the sum of all the numbers that are left.
```

**Review by SleeplessByte — APPROVED** [review]

File: exercises/sum-of-multiples/instructions.md (hunk 3 above)
Comment (kytrinyx) [reply exercises/sum-of-multiples/instructions.md]: Thanks for the tweaks, @SleeplessByte

File: exercises/sum-of-multiples/introduction.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+You work for a company that makes an online game called Enchanted Expeditions.
+
+The game is an adventure survival game.
+Players need to travel through treacherous terrain in search of valuable resources.
+As they explore, they collect materials like metals, fibers, and wood.
+These can be used to craft tools, survival gear, and weapons.
+
+However, crafting requires a special type of magical energy that is only awarded when a player has completed a level.
+
+The amount of energy awarded depends on which magical items the player found while exploring that level.
```
[the remark this answers is not in the document]
Comment (kytrinyx) [reply exercises/sum-of-multiples/introduction.md:12]: I think that works fine, actually. I'd like a second opinion, though, because I've been through so many iterations of this exercise that I'm worried I've lost the plot. @iHiD @ErikSchierboom thoughts?

File: exercises/sum-of-multiples/instructions.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# Instructions
+
+Your task is to write the code that calculates the energy points that get awarded to players when they complete a level.
+
+The points awarded depend on two things:
+
+- the level (as number) that the player completed
+- the base value of each magical item collected by the player during that level
```
Comment (IsaacG) [inline exercises/sum-of-multiples/instructions.md:8]:
```suggestion
- The level (a number) that the player completed.
- The base value of each magical item collected by the player during that level.
```

**Review by IsaacG — APPROVED** [review]

File: exercises/sum-of-multiples/introduction.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (iHiD) [reply exercises/sum-of-multiples/introduction.md:12]:
```suggestion
You work for a company that makes an online, fantasy-survival game.

When a player finishes a level, they are awarded energy points.
The amount of energy awarded depends on which magical items the player found while exploring that level.
```

[the remark this answers is not in the document]
Comment (iHiD) [reply exercises/sum-of-multiples/introduction.md:12]: I made it 5% less dry than @ee7's and I'm happy with this version - agree shortening slightly is good :)

[the remark this answers is not in the document]
Comment (kytrinyx) [reply exercises/sum-of-multiples/introduction.md:12]: Thank you!

File: exercises/sum-of-multiples/instructions.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+# Instructions
+
+Your task is to write the code that calculates the energy points that get awarded to players when they complete a level.
+
+The points awarded depend on two things:
+
+- the number of the level that the player completed
```
[the remark this answers is not in the document]
Comment (kytrinyx) [reply exercises/sum-of-multiples/instructions.md]: Yeah, I think they should. I'll fix that.

File: exercises/sum-of-multiples/instructions.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+1. For each magical item, take the base value and find all the multiples of that value that are less than or equal to the level number.
+2. Combine the sets of numbers.
+3. Remove any duplicates.
+4. Calculate the sum of all the numbers that are left.
+
+Let's look an example:
+
+**The player completed level 20 and found two magical items with base values of 3 and 5.**
+
+To calculate the energy points earned by the player, we need to find all the unique multiples of these base values that are less than or equal to level 20.
```
Comment (IsaacG) [inline exercises/sum-of-multiples/instructions.md:21]: @kytrinyx We changed the problem definition in the markdown and it no longer matches the tests.

```diff
- Multiples of 5 less than 20: 5, 10, 15
+ Multiples of 5 up to 20: `{5, 10, 15, 20}`
```

@vaeng is going to open a PR to fix this.

Comment (IsaacG) [reply exercises/sum-of-multiples/instructions.md:21]: See #2261
